dashboard/src/components/Skeleton/Skeleton.tsx

```diff
@@ -1,14 +1,16 @@
-import type { ReactNode } from 'react';
+import type { ReactNode, ComponentProps } from 'react';
 
 import { Skeleton as SkeletonComponent } from '@/components/ui/skeleton'; //to avoid name conflict we rename the import
 
-type SkeletonProps = {
+type SkeletonProps = ComponentProps<typeof SkeletonComponent> & {
   children: ReactNode;
 };
 
-const Skeleton = ({ children }: SkeletonProps): JSX.Element => {
+const Skeleton = ({ children, className }: SkeletonProps): JSX.Element => {
   return (
-    <SkeletonComponent className="grid h-[400px] place-items-center">
+    <SkeletonComponent
+      className={`grid h-[400px] place-items-center ${className}`}
```

Comment:
we are using a library `classNames` that helps us to concat different classNames without needing to use `$something`. here is an example, but you can search for more in the code

```javascript
className={classNames('flex flex-row gap-2 pb-1', hasBorder)}
```

Reply:
don't use classnames, use our `cn` library